### ISSUE TITLE
0.2.8 Fails to run auto-genned project

### ISSUE THREAD

**Issue by Gerkinfeltser** [GoranM/bdx#1040]
Attempting to run the auto-generated Sacky project produces this error when hitting "Export and Run BDX" or hitting "P" in the viewport. This is with the 0.2.8 release.

```
Traceback (most recent call last):
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\ops\exprun.py", line 285, in execute
    export(self, context, bpy.context.scene.bdx.multi_blend_export, bpy.context.scene.bdx.diff_export)
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\ops\exprun.py", line 243, in export
    raise export_error
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\ops\exprun.py", line 132, in export
    bpy.ops.export_scene.bdx(filepath=file_path, scene_name=scene.name, exprun=True)
  File "C:\Program Files\Blender Foundation\Blender\2.79\scripts\modules\bpy\ops.py", line 189, in __call__
    ret = op_call(self.idname_py(), None, kw)
RuntimeError: Error: Traceback (most recent call last):
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\exporter.py", line 918, in execute
    return export(context, self.filepath, self.scene_name, self.exprun, self.apply_modifier)
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\exporter.py", line 865, in export
    generate_bitmap_fonts(fonts, fontgen_dir, fonts_dir, textures_dir);
  File "C:\Users\user\AppData\Roaming\Blender Foundation\Blender\2.79\scripts\addons\bdx\exporter.py", line 744, in generate_bitmap_fonts
    (try running the game without any text objects first)")
Exception: Font gen: Can't find required gdx jars                 (try running the game without any text objects first)

location: C:\Program Files\Blender Foundation\Blender\2.79\scripts\modules\bpy\ops.py:189



location: <unknown location>:-1


```

**Comment by rafcolson** [comment]
Did you specify a valid java path?

**Comment by Gerkinfeltser** [comment]
Typing java in a cmd prompt yeilded non-error like results but I'll triple check.

**Comment by Gerkinfeltser** [comment]
Yeah, same issue with java 7 jdk. 👎

**Comment by rafcolson** [comment]
Did you specify a valid java path in the java path field before creating the project?

**Comment by Gerkinfeltser** [comment]
Think so. Pretty much filling it out like this (not actually my website). Is this correct?
![blender_2018-03-28_16-46-12](https://user-images.githubusercontent.com/1477787/38055462-c59d24d2-32a7-11e8-9a0c-6921a04046f4.png)

**Comment by rafcolson** [comment]
Oh. I'm sorry. I wasn't thinking. Please ignore my earlier comment.

I just tested the 0.2.8 release and I'm having issues too.

Maybe try my own build of [BDX v0.2.8](https://www.dropbox.com/s/qm0i8w9uo1kgvvv/bdx.zip?dl=0), because this works for me.

**Comment by rafcolson** [comment]
[This issue comment](https://github.com/GoranM/bdx/issues/568#issuecomment-191985495) might tell us more about the error message. Some gdx jars could not be found so my guess is that there is an issue with your gradle.

Since I'm not understanding everything about this, I hope @GoranM or @SolarLune could take it from here. I'm sorry I couldn't be of much help.

**Comment by Gerkinfeltser** [comment]
Thanks so much for your help so far rafcolson! Still getting issues though I think it may be some issue with my java/jdk installation. Running `gradlew.bat desktop:dist` causes and error due to java not being able to find tools.jar. I'm rewatching Goran's installation video to make sure I'm doing java stuff correctly. :) I'll report back.

**Comment by Gerkinfeltser** [comment]
Aight, it was indeed something weird with my jdk installation. Removing all jdk & java related stuff & then reinstalling (jdk 1.7) seems to have done the trick. Thanks again for the assistance rafcolson!

**Comment by rafcolson** [comment]
Glad it worked out.

**Comment by Gerkinfeltser** [comment]
Actually I had this same issue arise on another computer I attempted to run BDX 0.2.8 on as well.

I'm wondering if it isn't because I'm leaving the android SDK field blank when I make a new project. This seems to upset gradlew/java/blender/a higher power and throw the original error. Commenting out the android stuff in the project dir's build.gradle & removing the 'android' bit from settings.gradle in the same folder seems to fix the issue and allow BDX to complete a compile. 

I marked this as open again as I thought it might be a good idea to set BDX to auto-remove/ignore the android stuff if the android SDK field is left blank when building a new project. Perhaps this isn't a smart idea though. I am merely a hobbyist programmer. :)
